src/docfx/lib/path/BasePath.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Docs.Build
+{
+    [JsonConverter(typeof(BasePathJsonConverter))]
+    internal class BasePath
```

Comment:
```suggestion
    internal struct BasePath
```

Reply:
Line 219 in `Config.cs` we are changing the value of `Original`